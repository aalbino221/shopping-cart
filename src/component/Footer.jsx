/* eslint-disable jsx-a11y/anchor-is-valid */
import React from 'react';

export default function Footer() {
  const style = {
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
    fontSize: '1.5rem',
    padding: '3rem',
  };

  return (
    <div className="footer" style={style}>
      <a
        href="https://github.com/aalbino221"
        style={{ textDecoration: 'none', color: 'black' }}
      >
        <i className="fa-brands fa-github" style={{ marginRight: '0.2rem' }} />
        aalbino221
      </a>
    </div>
  );
}
