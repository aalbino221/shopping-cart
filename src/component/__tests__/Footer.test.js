import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom'; // optional
import Footer from '../Footer';

describe('Footer Component', () => {
  it('Renders Correctly', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
