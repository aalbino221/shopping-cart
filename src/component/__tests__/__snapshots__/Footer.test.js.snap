// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Renders Correctly 1`] = `
<div>
  <div
    class="footer"
    style="display: flex; justify-content: center; align-items: center; font-size: 1.5rem; padding: 3rem;"
  >
    <a
      href="https://github.com/aalbino221"
      style="text-decoration: none; color: black;"
    >
      <i
        class="fa-brands fa-github"
        style="margin-right: 0.2rem;"
      />
      aalbino221
    </a>
  </div>
</div>
`;
